app/services/determine_training_record_state.rb

```diff
@@ -2,26 +2,156 @@
 
 # noinspection RubyTooManyMethodsInspection, RubyTooManyInstanceVariablesInspection, RubyInstanceMethodNamingConvention
 class DetermineTrainingRecordState < BaseService
+  attr_reader :participant_profile_id, :school_id, :appropriate_body_id, :delivery_partner_id, :induction_record_id
+
+  # enum validation_state: {
```

Comment:
did we need to convert the rest of the  enums or happy to remove completely?

Reply:
I think having them is going to be useful even if it's just in a documentation capacity; it's not going to be immediately clear what the possible values for each state are. I'll reformat it into a comment with a bit of an explanation